addons/survey/views/survey_templates_statistics.xml

```diff
@@ -247,48 +247,22 @@
-                    </tr>
-                </thead>
-                <tbody>
-                    <t t-foreach="non_skipped_records" t-as="input_line">
-                        <tr t-att-class="'d-none' if not input_line in non_skipped_records[:page_record_limit] else ''">
-                            <td>
-                                <t t-if="no_print_url"><t t-esc="input_line_index + 1"></t></t>
-                                <t t-else="">
-                                    <a t-att-href="input_line.user_input_id.get_print_url()">
-                                        <t t-esc="input_line_index + 1"></t>
-                                    </a>
-                                </t>
-                            </td>
-                            <td>
-                                <!-- If answer already covered by filter or if there is only one line to display, do not allow filtering on it again -->
-                                <t t-if="question.id in already_filtered_questions or len(table_data) == 1">
-                                    <t t-esc="input_line._get_answer_value()"/>
-                                </t>
-                                <t t-else="">
-                                    <div class="d-flex justify-content-between">
-                                        <t t-esc="input_line._get_answer_value()"/>
-                                        <a class="text-primary filter-add-answer d-print-none"
-                                            data-model-short-key="L" t-att-data-record-id="input_line.id"
-                                            role="button" aria-label="Filter surveys" title="Only show survey results having selected this answer">
-                                            <i class="fa fa-filter"/>
-                                        </a>
-                                    </div>
-                                </t>
-                            </td>
+        <t t-set="records_json" t-value="json.dumps([[input_line.id, input_line._get_answer_value(), input_line.user_input_id.get_print_url()] for input_line in list(filter(lambda record: not record.skipped, table_data))])"/>
+        <t t-set="hide_filter" t-value="question.id in already_filtered_questions or len(table_data) == 1"/>
+        <div class="survey_table_with_pagination" t-att-data-answers-json="records_json" t-att-data-hide-filter="hide_filter" t-att-data-record_limit="page_record_limit">
+            <div class="o_survey_results_table_wrapper overflow-auto">
+                <table class="table table-hover table-sm o_survey_results_table_indexed" t-att-id="'survey_table_question_%d' % question.id">
+                    <thead>
+                        <tr>
+                            <th>#</th>
+                            <th>User Responses</th>
                         </tr>
-                    </t>
-                </tbody>
-            </table>
+                    </thead>
+                    <tbody/>
```

Comment:
Perhaps we should add a comment specifying that it will be rendered by an interaction using `survey.pagination_entry`? Perhaps better named `survey.paginated_result_text_rows` ?